web/frontend/src/components/slam.vue

```diff
@@ -117,6 +117,35 @@ const fetchSLAMPose = (name: string): Promise<commonApi.Pose> => {
   });
 };
 
+const fetchFeatureFlags = (name: string): Promise<{[key: string]: boolean}> => {
+  return new Promise((resolve, reject): void => {
+    const request = new commonApi.DoCommandRequest();
+    request.setName(name);
+    request.setCommand(Struct.fromJavaScript({ feature_flag: true }));
+    props.client.slamService.doCommand(
+      request,
+      new grpc.Metadata(),
+      (error: ServiceError|null, responseMessage: commonApi.DoCommandResponse|null) => {
+
+        /*
+         * Note: we ignore unimplementedError because in the current implementation it
+         *  signifies that the feature flag is false
+         */
+        if (error) {
+          if (error.code !== grpc.Code.Unimplemented) {
+            reject(error);
+            return;
+          }
+          resolve({});
```

Comment:
[nit] I think it's slightly cleaner to use `if (condition)` rather than `if (!condition)`, like:

```
if (error.code == grpc.Code.unimplemented) {
    // <comment>
    resolve({});
    return;
}
reject(error);
return;
```